### PR TITLE
Feature/nsa 5308 name text can overlap

### PR DESCRIPTION
**Pull request by MarekArtur**

### CODE REVIEW

File: config/login.dfe.support.dev.json [hunk 1]
```diff
@@ -26,55 +42,151 @@
...
+      "schema": "dbo",
+      "pool": {
+        "max": 100,
+        "min": 0,
+        "acquire": 30000,
+        "idle": 10000
+      }
     },
-    "cacheConnectionString": "redis://localhost:6379"
+    "cacheConnectionString": "redis://:BUeq3HkJzVvPRTLeNhl8MgLrFF+gLUkgvboHrwLMCF4=@s141d01-signin-shd-rc.redis.cache.windows.net:6380/7?tls=true"
```
Comment (AMcCartneyMTH) [inline config/login.dfe.support.dev.json:109]: Should this not be running against a local Redis instance?

File: config/login.dfe.support.dev.json [hunk 2]
```diff
@@ -26,55 +42,151 @@
...
   },
   "serviceMapping": {
     "type": "redis",
     "params": {
-      "connectionString": "redis://localhost:6379"
+      "connectionString": "redis://:BUeq3HkJzVvPRTLeNhl8MgLrFF+gLUkgvboHrwLMCF4=@s141d01-signin-shd-rc.redis.cache.windows.net:6380/3?tls=true"
```
Comment (AMcCartneyMTH) [inline config/login.dfe.support.dev.json:114]: As above in terms of target instance.

File: config/login.dfe.support.dev.json [hunk 3]
```diff
@@ -26,55 +42,151 @@
...
+    "service": {
+      "url": "https://s141d01-signin-dev.azurewebsites.net",
+      "auth": {
+        "type": "aad",
+        "tenant": "platformidentity.onmicrosoft.com",
+        "authorityHostUrl": "https://login.microsoftonline.com/9c7d9dd3-840c-4b3f-818e-552865082e16",
+        "clientId": "350e8e5d-3875-44da-878d-1bb7feb7125d",
+        "clientSecret": "Fj=owAV=m9@477wzLVgNPo]VT?4YZUnW",
+        "resource": "b220f0ce-ad01-49c2-aa72-379aec0cb096"
+      }
```
Comment (AMcCartneyMTH) [inline config/login.dfe.support.dev.json:129]: I'd like to clarify that this configuration should be present on the integration branch.

File: config/login.dfe.support.dev.json [hunk 4]
```diff
@@ -26,55 +42,151 @@
...
+    "usersDiff": "*/10 2-23 * * *",
+    "userDevices": "*/15 2-23 * * *",
+    "accessRequests": "*/15 2-23 * * *",
+    "indexTidy": "*/15 2-23 * * *"
   },
   "toggles": {
     "useGenericAddUser": true
+  },
+  "notifications": {
+    "connectionString": "redis://:BUeq3HkJzVvPRTLeNhl8MgLrFF+gLUkgvboHrwLMCF4=@s141d01-signin-shd-rc.redis.cache.windows.net:6380/4?tls=true"
```
Comment (AMcCartneyMTH) [inline config/login.dfe.support.dev.json:186]: This appears to be your dev secret?

File: package.json [hunk 5]
```diff
@@ -51,7 +51,7 @@
     "login.dfe.winston-appinsights": "git+https://github.com/DFE-Digital/login.dfe.winston-appinsights.git#v3.0",
     "moment": "^2.29.1",
     "node-schedule": "^1.3.3",
-    "openid-client": "^1.20.0",
+    "openid-client": "^5.1.2",
```
Comment (AMcCartneyMTH) [inline package.json:54]: This seems like a significant version bump. Can we confirm that regression has been performed?

File: package.json [hunk 6]
```diff
@@ -61,12 +61,12 @@
...
-    "eslint": "^6.8.0",
+    "eslint": "^8.7.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.23.0",
-    "ioredis-mock": "2.4.1",
-    "jest": "^24.9.0",
-    "jest-cli": "^24.9.0",
+    "ioredis-mock": "^6.0.0",
+    "jest": "^27.4.7",
+    "jest-cli": "^27.4.7",
```
Comment (AMcCartneyMTH) [inline package.json:69]: As above regarding regression testing of packages.

File: src/app/accessRequests/views/request.ejs [hunk 7]
```diff
@@ -3,7 +3,7 @@
 
 <dl class="rows">
     <dt>Name</dt>
-    <dd><%=locals.accessRequest.name%></dd>
+    <dd ><%=locals.accessRequest.name%></dd>
```
Comment (AMcCartneyMTH) [inline src/app/accessRequests/views/request.ejs]: Can we remove the space please?

**Review by AMcCartneyMTH — COMMENTED** [review]
Some comments around versioning.

File: src/app/accessRequests/views/request.ejs (hunk 7 above)
Comment (MarekArtur) [reply src/app/accessRequests/views/request.ejs]: yes

**Comment by AMcCartneyMTH** [comment]
Replaced by subsequent PR

**Pull request closed without merging**